style.css

```diff
@@ -82,29 +82,28 @@ img {
 	background: #f4f5f6;
```

Comment:
It looks okay in my IDE. I only added a few lines, would you like me to auto-reformat?